src/main/kotlin/org/kamiblue/client/gui/rgui/component/Slider.kt

```diff
@@ -114,7 +114,9 @@ open class Slider(
                     (renderHeight * ClickGUI.getScaleFactor()).roundToInt()
             )
         }*/
-        FontRenderAdapter.drawString(name, 1.5f, 1.0f, color = GuiColors.text)
+        val text = if (isBold()) "§l $name" else name
```

Comment:
Is there no way to do this without hardcoded magic values? At least use `if (isBold()) ${TextFormatting.BOLD format name} else name`

Reply:
You might be able to do that, I didn't know of that function, will check now.